code/obj/artifacts/artifact_objects/flinger.dm

```diff
@@ -0,0 +1,118 @@
+/obj/artifact/flinger
+	name = "artifact flinger"
+	associated_datum = /datum/artifact/flinger
+
+	throw_impact(atom/A, datum/thrown_thing/thr)
+		. = ..(A, thr)
+		if(iscarbon(A))
+			var/mob/living/carbon/unit = A
+			if(thr.bonus_throwforce >= 20)
+				unit.emote("scream")
+			if(thr.bonus_throwforce >= 90)
+				unit.gib()
```

Comment:
I'm not sure adding the possibility of being instantly gibbed for activating an artifact is a good idea, even cyborg converters allow the possibility of being saved or just playing as a borg. Maybe instead throw the *victim* with `THROW_THROUGH_WALL`?